### ISSUE TITLE
Payment Details

### ISSUE THREAD

**Issue by reibanaag** [nozzlegear/ShopifySharp#902]
Is there any reason why the payment details are not included in the response?

![image](https://github.com/nozzlegear/ShopifySharp/assets/77881215/bf1773b7-ada2-490a-8f06-e97e3fdc2ef5)

As our customer require us to pull this details?

Will this details be returned in the future?

**Comment by nozzlegear** [comment]
Hey @reibanaag! Which class isn't including the payment details in the response? Most likely the payment details weren't available at the time the class was implemented in ShopifySharp, so it probably just needs to be added now that they are available.

**Comment by reibanaag** [comment]
@nozzlegear Yes on the response. As described on this https://shopify.dev/docs/api/admin-rest/2023-07/resources/order#resource-object. Event the response call via REST also includes the payment details. I hope it can be added to the next iteration. Thank you.
